Sources/SemanticIndex/UpdateIndexStoreTaskDescription.swift

```diff
@@ -475,29 +634,89 @@ package struct UpdateIndexStoreTaskDescription: IndexTaskDescription {
       // Indexing will frequently fail if the source code is in an invalid state. Thus, log the failure at a low level.
       logger.debug(
         """
-        Updating index store for \(indexFile.forLogging) terminated with non-zero exit code \(code)
+        Updating index store for terminated with non-zero exit code \(code) for \(indexFiles)
         Stderr:
         \(stderr)
         Stdout:
         \(stdout)
         """
       )
-      BuildSettingsLogger.log(level: .debug, settings: buildSettings, for: indexFile)
+    // BuildSettingsLogger.log(level: .debug, settings: buildSettings, for: indexFile)
```

Comment:
`.first` like above (in which case, maybe add another overloads that takes a list and does that and then use https://github.com/swiftlang/sourcekit-lsp/pull/2293/files#diff-ea45e11d345f3369fa911535dbd114c644d1757f43bbc4e726f644757da2bd83R388 as well)

Reply:
Good catch. Added an overload that takes a list